types/read-time-estimate/index.d.ts

```diff
@@ -0,0 +1,28 @@
+// Type definitions for read-time-estimate 0.0
+// Project: https://github.com/pritishvaidya/read-time-estimate#readme
+// Definitions by: rubek <https://github.com/rubek-joshi>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/*~ You can declare types that are available via importing the module */
+declare function readTimeEstimate(
+    string: string,
+    customWordTime?: number,
+    customImageTime?: number,
+    chineseKoreanReadTime?: number,
+    imageTags?: string[],
+): readTimeEstimate.ReadTimeEstimate;
+
+declare namespace readTimeEstimate {
+    interface ReadTimeEstimate {
+        humanizedDuration: string;
+        duration: number;
+        totalWords: number;
+        wordTime: number;
+        totalImages: number;
+        imageTime: number;
+        otherLanguageTimeCharacters: number;
+        otherLanguageTime: number;
+    }
+}
+
+export = readTimeEstimate;
```

Comment:
It looks like the underlying package is set up so that it exports this function through a `default` property, at least via the regular `main` entry point in its `package.json` if you're not using any bundler. The documentation in the readme of the package doesn't really reflect that, unfortunately. So to accurately reflect that on the type declarations, I think you should change this to use a default export, i.e. `export default function readTimeEstimate(...)`, and `export interface ReadTimeEstimate`.
Then you can also update the import on the test file.

Reply:
Thanks for nudging me in the right direction, I have changed the export statement.